oshi-core/src/main/java/oshi/software/os/unix/aix/AixOSProcess.java

```diff
@@ -349,17 +356,19 @@ static State getStateFromOutput(char stateValue) {
         State state;
         switch (stateValue) {
         case 'O':
+            state = INVALID;
+            break;
+        case 'A':
             state = RUNNING;
             break;
-        case 'S':
-            state = SLEEPING;
+        case 'I':
+            state = NEW;
             break;
-        case 'R':
         case 'W':
-            state = WAITING;
+            state = WAITING; //need confirmation on this
```

Comment:
`man ps` has a few more letters here.  I'd bring back the `R` for running and `S` for sleeping.  I think "swapped" might be sleeping rather than waiting?

            For processes:
              O                   Nonexistent
              A                   Active
              W                   Swapped
              I                   Idle (waiting for startup)
              Z                   Canceled
              T                   Stopped
            For kernel threads:
              O                   Nonexistent
              R                   Running
              S                   Sleeping
              W                   Swapped
              Z                   Canceled
              T                   Stopped

Reply:
Yeah i saw those but haven't added states relevant to thread. Will add those when i work on thread details.